### DISCUSSION TITLE
Allow ammunition boxes to be affected by quality.

### DISCUSSION THREAD

**Discussion by r4d6** [FakeFishGames/Barotrauma#7713] [Ideas and suggestions]
Ammo boxes currently work by using the durability as a sort of ammo counter, which is the same thing that welding fuel, oxygenite & incendium tanks & fuel rods use. 

And just like the aforementioned items, they can be recycled into an intact version in a fabricator for the same overall material cost than if you deconstructed the box and made a new one.

As such, since 4 other items that work on a similar principle are affected by the quality system, which give them more durability, and thus  a longer usage, it only make sense to allow the ammo boxes to be affect by quality too. A quality-increased effect could be appended to a Security officer's talent like 'Tandem Fire' or, more in theme, 'Munitions Expertise'.
It would also give a reason for securities officer to make more ammo boxes beyond just the time it take when not meeting the weapon skill requirement.

The simplest implementation of the Quality System would be to increase the box's durability, same as the fuel rods and various tanks, which should translate into more shots per box, which is still a good thing to have.

**Comment by gerakis** [comment]
1) bullets wont stack, which would be annoying
2) bullets cannot be taken apart for the same resources it took to make them, making problem 1 impossible to fix

**Reply by r4d6** [reply]
1. Only different-quality bullets wouldn't be able to be transferred to one-another, which would make it slightly less annoying. 
2. [Yes they can](https://barotraumagame.com/wiki/Coilgun_Ammunition_Box)

EDIT : Rereading your comment, it seems you are talking about **revolver, handcannon & shotgun** ammo, which is _not_ what this post is about. I am talking about the ammo boxes for the Coilgun, Pulse Laser & Chaingun, which can't be stacked anyway, I am **not** talking about stuff like hand-weapon ammo.
